src/Microsoft.Azure.WebJobs.Host/Blobs/Listeners/StorageAnalyticsLogEntry.cs

```diff
@@ -139,23 +137,19 @@ public BlobPath ToBlobPath()
             else
             {
                 // assuming key is "/account/container/blob"
-                int startPos = RequestedObjectKey.Length > 1 ? RequestedObjectKey.IndexOf('/', 1) : -1;                
+                int startPos = RequestedObjectKey.Length > 1 ? RequestedObjectKey.IndexOf('/', 1) : -1;
                 path = startPos != -1 ? RequestedObjectKey.Substring(startPos + 1) : String.Empty;
             }
 
             if (String.IsNullOrEmpty(path))
             {
-                throw new FormatException("Failed to parse RequestedObjectKey property of the log entry. " +
-                    "It should be in one of the supported formats: " +
-                    @"""https://account.blob.core.windows.net/container/blob"", or" +
-                    @"""/account/container/blob""");
+                return null;
```

Comment:
I think we should update the parse code to take into account StorageAnalyticsLogColumnId.RequestStatus. In the repro case, this will be a ThrottlingError. We could then change consumer code so it only processes sucessfull requests. I.e. change BlobLogListener to include entry.IsSuccess in its where clause. Thoughts?